hack/util.sh

```diff
@@ -114,8 +114,10 @@ function wait_for_url {
   set_curl_args $wait $times
 
   set +e
+  cmd="env -i CURL_CA_BUNDLE=${CURL_CA_BUNDLE:-} $(which curl) ${clientcert_args} -fs ${url}"
+  #echo "run: ${cmd}"
   for i in $(seq 1 $times); do
-    out=$(curl ${clientcert_args} -fs $url 2>/dev/null)
+    out=$(${cmd})
```

Comment:
Actually... it is, because we want to know when something crash burns, and -f -s also reduce the amount of crap.

----- Original Message -----
> >    for i in $(seq 1 $times); do
> > -    out=$(curl ${clientcert_args} -fs $url 2>/dev/null)
> > +    out=$(${cmd})
> 
> lost the stderr redirect... was that intentional?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/openshift/origin/pull/1436/files#r27126973